weaviate/values.yaml

```diff
@@ -13,6 +13,46 @@ image:
   pullPolicy: IfNotPresent
   pullSecrets: []
 
+experimental:
+  querier:
+    enabled: false
+    image: semitechnologies/weaviate-experimental:main-b5da000
+    storage_size: 10Gi
+    pullPolicy: IfNotPresent
+    command: ["/bin/weaviate"]
+    args:
```

Comment:
What happens if we don't pass these args? I mean, could we provide some default values so that the querier would work out of the box?  Mostly to improve the user experience and avoid requesting the user who deploys to know about the weaviate's architecture internals. I know this is a first PR to test the new querier, but for example, the contextionary endpoint should be added conditionally if contextionary is configured only. Otherwise it will fail when it won't be enabled as a module.
Also, the monitoring seems a bit redundant, don't we have an env var when we want monitoring to be working (PROMETHEUS_BLABLA_ENABLED)?  we should enable this only if that env var is enabled, imho

Reply:
> could we provide some default values so that the querier would work out of the box? 

Absolutely we should. I thought [I had some sane default values](https://github.com/weaviate/weaviate/blob/main/exp/query/config.go#L14-L28). But that doesn't work well with our helm chart values. I will update it 👍 

> I know this is a first PR to test the new querier, but for example, the contextionary endpoint should be added conditionally if contextionary is configured only. Otherwise it will fail when it won't be enabled as a module.

Currently `querier` has hardcoded dependency with `contextionary` unfortunately. We will remove it in the future and I will update the conditional loading of modules. Very good catch.

> Also, the monitoring seems a bit redundant, don't we have an env var when we want monitoring to be working (PROMETHEUS_BLABLA_ENABLED)? we should enable this only if that env var is enabled, imho

Good point. Currently `querier` doesn't respect the ENV `PROMETHEUS_MONITORING_ENABLED`. I think we should (I can take it up in the follow up PR if that's ok?). 

Having said that I have some opinions about configs :) I like to keep this cli-flags as well. IMHO cli-flags should take highest priority than ENVs (and every config should be able to pass via cli-flags in addition to ENVs). Not a big fan of ENV based configs tbh, in my experience it's hard to manage at scale (10s is fine, 100s is crazy). Prefer cli or file based config instead. Main rationale is I want my configs to be "local" and "explicit". ENVs are by design "global" and "implicit" (any other process that share same pod, shell can set different values).

Reply:
> Absolutely we should. I thought [I had some sane default values](https://github.com/weaviate/weaviate/blob/main/exp/query/config.go#L14-L28). But that doesn't work well with our helm chart values. I will update it 👍

I think we still cannot provide sane default for some flags like `schema_url`, `contextionary_url` and `minio_url`. Because this is totally different when running in k8s, all those will be local `service` name.

Reply:
I removed some args to make it work with defaults.

This PR also changes port mapping 9091 (used by prometheus server in our setup) to 7071
https://github.com/weaviate/weaviate/pull/6043

Reply:
Done.